.github/RssRegression.java

```diff
@@ -0,0 +1,182 @@
+//usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS com.ibm.cos:ibm-cos-java-sdk:2.1.0
+//DEPS javax.xml.bind:jaxb-api:2.4.0-b180830.0359
+
+package io.quarkus;
+
+import com.ibm.cloud.objectstorage.ClientConfiguration;
+import com.ibm.cloud.objectstorage.SDKGlobalConfiguration;
+import com.ibm.cloud.objectstorage.auth.AWSCredentials;
+import com.ibm.cloud.objectstorage.auth.AWSStaticCredentialsProvider;
+import com.ibm.cloud.objectstorage.client.builder.AwsClientBuilder.EndpointConfiguration;
+import com.ibm.cloud.objectstorage.oauth.BasicIBMOAuthCredentials;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3;
+import com.ibm.cloud.objectstorage.services.s3.AmazonS3ClientBuilder;
+import com.ibm.cloud.objectstorage.services.s3.model.AmazonS3Exception;
+import com.ibm.cloud.objectstorage.services.s3.model.GetObjectRequest;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import static java.lang.System.exit;
+
+public class RssRegression {
+
+    private static AmazonS3 _cosClient;
+    private static String ENDPOINT_URL;
+    private static String LOCATION;
+
+    static {
+        ENDPOINT_URL = "https://s3.eu-gb.cloud-object-storage.appdomain.cloud";
+        SDKGlobalConfiguration.IAM_ENDPOINT = "https://iam.cloud.ibm.com/identity/token";
+        LOCATION = "eu-gb";
+    }
+
+    /**
+     * @param args
+     */
+    public static void main(String[] args) {
+
+        if(args.length != 6){
```

Comment:
fyi you can greatly simplify the parsing and usage text by using picocli ... not a blocker for the PR but just a pointer.  example: https://github.com/maxandersen/jbang/blob/master/examples/smee.java#L51

Reply:
Thanks for the pointer